stories/components/footer/footer.html.twig

```diff
@@ -37,50 +37,48 @@
       </div>
       <div class="bix-site-footer__column bix-site-footer__column--accreditations">
         <h3 class="bix-site-footer__title">Accreditations</h3>
-          <ul >
-            <li>
-              <a
-                href="https://www.sba.gov/business-guide/grow-your-business/women-owned-businesses"
-                rel="noopener noreferrer"
-                target="_blank">
-                  Woman-owned small business
-              </a>
-            </li>
-            <li>
-              <a
-                href="https://www.iso.org/standard/62085.html"
-                rel="noopener noreferrer"
-                target="_blank">
-                ISO 9001:2015
-              </a>
-            </li>
-            <li>
-              <a
-                href="https://cmmiinstitute.com/learning/appraisals/levels"
-                rel="noopener noreferrer"
-                target="_blank">
-                CMMI Level 3
-              </a>
-            </li>
-          </ul>
-          <div class="accreditations-logos">
+        <ul >
+          <li>
+            <a
+              href="https://www.sba.gov/business-guide/grow-your-business/women-owned-businesses"
+              rel="noopener noreferrer"
+              target="_blank">
+              Woman-owned small business
+            </a>
+          </li>
+          <li>
             <a
               href="https://www.iso.org/standard/62085.html"
               rel="noopener noreferrer"
-              target="_blank"
-              title="ISO certified logo">
-              <img
-                src="https://bixal.com/static/7110e309c7dc3b7c2aa59397bde3793a/iso-certified.svg"
-                alt="">
+              target="_blank">
+              ISO 9001:2015
             </a>
+          </li>
+          <li>
             <a
               href="https://cmmiinstitute.com/learning/appraisals/levels"
               rel="noopener noreferrer"
-              target="_blank"
-              title="Capability Maturity Model Integration Level 3 logo">
-              <img src="https://bixal.com/static/9f64d5381d0ca7afecdd062b1bac2a68/cmmi.svg" alt="">
+              target="_blank">
+              CMMI Level 3
             </a>
-          </div>
+          </li>
+        </ul>
+        <div class="accreditations-logos">
+          <a
+            href="https://www.iso.org/standard/62085.html"
+            rel="noopener noreferrer"
+            target="_blank">
+            <img
+              src="https://bixal.com/static/7110e309c7dc3b7c2aa59397bde3793a/iso-certified.svg"
+              alt="ISO certified logo">
+          </a>
+          <a
+            href="https://cmmiinstitute.com/learning/appraisals/levels"
+            rel="noopener noreferrer"
+            target="_blank">
+            <img src="https://bixal.com/static/9f64d5381d0ca7afecdd062b1bac2a68/cmmi.svg" alt="Capability Maturity Model Integration Level 3 logo">
+          </a>
+        </div>
```

Comment:
Ideally, these would be under their respective links and given an `aria-hidden="true"` because they're decorative.

Outside the scope of this work though.